rendering_resource_manager_service/config/management/rendering_resource_settings_manager.py

```diff
@@ -82,6 +83,7 @@ def update(cls, params):
             settings.scheduler_rest_parameters_format = \
                 str(params['scheduler_rest_parameters_format'])
             settings.graceful_exit = params['graceful_exit']
+            settings.wait_until_running = params['wait_until_running']
```

Comment:
Actually, the str2bool is not necessary since graceful_exit and wait_until_running should be booleans in the first place. In the curl example, they are passed as string and this is why a string-to-boolean conversion is needed. I removed the conversion.